site/profile/manifests/windows/registry.pp

```diff
@@ -0,0 +1,43 @@
+class profile::windows::registry
+(
+  String $ntpserver = "time.windows.com",
+)
+
+{
+
+#Puppet module for managing base Windows registry settings
+
+# Disable UAC
+  registry_value { 'HKLM\Software\Microsoft\Windows\CurrentVersion\Policies\System\EnableLUA':
+    ensure => 'present',
+    data   => ['0'],
+    type   => 'dword',
+  }
+
+# Configure NTP Server
+  registry_value { 'HKLM\SYSTEM\Currentcontrolset\Services\W32time\Parameters\NtpServer':
+    ensure => 'present',
+    data   => ["$ntpserver,0x9"],
+    type   => 'string',
+  }
+
+# Disable Windows Firewall
```

Comment:
Would it be better instead to merely add Allow All firewall rules instead of disabling the firewall?  This means you can still log packets etc and not incur the wrath of constantly being reminded to turn the FW back on.